src/core/org/apache/jmeter/gui/MainFrame.java

```diff
@@ -680,7 +683,15 @@ private void addQuickComponentHotkeys(JTree treevar) {
 
             @Override
             public void actionPerformed(ActionEvent actionEvent) {
-                String propname = "gui.quick_" + actionEvent.getActionCommand();
+                //Bug 62336
+                AWTEvent current_event = EventQueue.getCurrentEvent();
+                String key_text = "";
+                if(current_event instanceof KeyEvent) {
+                    KeyEvent key_event = (KeyEvent)current_event;
+                    key_text = KeyEvent.getKeyText( key_event.getKeyCode() );
```

Comment:
Again a minor nit: No space needed after the opening and before the closing parenthesis as this is a function call.
And as above variable names should be written in camel case: `keyEvent` instead of `key_event`.